presto-cli/src/main/java/io/prestosql/cli/AlignedTablePrinter.java

```diff
@@ -186,6 +196,24 @@ static String formatHexDump(byte[] bytes)
         return HEX_BYTE_JOINER.join(createHexPairs(bytes));
     }
 
+    static String formatList(List<? extends Object> list)
+    {
+        ImmutableList.Builder<Object> formattedList = ImmutableList.builder();
+
+        list.forEach(value -> formattedList.add(formatValue(value)));
+
+        return formattedList.build().toString();
```

Comment:
I think it would be better to explicitly format the list:
```
return list.stream()
 .map(AlignedTablePrinter::formatValue)
 .collect(joining(", ", "[", "]));
```